relay-server/server/relayServer.go

```diff
@@ -663,12 +672,14 @@ func connectToKubeArmor(nodeIP, port string) error {
 
 // GetFeedsFromNodes Function
 func (rs *RelayServer) GetFeedsFromNodes() {
+
 	rs.WgServer.Add(1)
 	defer rs.WgServer.Done()
 
 	if K8s.InitK8sClient() {
 		kg.Print("Initialized the Kubernetes client")
 
+		ClientListLock.RLock()
```

Comment:
again we dont need a mutex lock here 
this is a infinite for loop the lock will not unlock until we stop the relay server itself